inference-engine/src/low_precision_transformations/src/move_fake_quantize.cpp

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/move_fake_quantize.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/opsets/opset1.hpp>
+
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pattern/op/or.hpp>
+
+#include "low_precision/network_helper.hpp"
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::low_precision::MoveFakeQuantize, "MoveFakeQuantize", 0);
+
+MoveFakeQuantize::MoveFakeQuantize(const Params& params) : LayerTransformation(params) {
+    auto concat = ngraph::pattern::wrap_type<opset1::Concat>(pattern::consumers_count(1));
+    auto operation = ngraph::pattern::wrap_type<opset1::Relu>({ concat });
+    auto input_low = ngraph::pattern::wrap_type<ngraph::opset1::Constant>();
+    auto input_high = ngraph::pattern::wrap_type<ngraph::opset1::Constant>();
+    auto output_low = ngraph::pattern::wrap_type<ngraph::opset1::Constant>();
+    auto output_high = ngraph::pattern::wrap_type<ngraph::opset1::Constant>();
+    auto fq_with_operation = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ operation,
+        input_low,
+        input_high,
+        output_low,
+        output_high});
+    auto fq = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ concat,
+        input_low,
+        input_high,
+        output_low,
+        output_high });
+
+    ngraph::graph_rewrite_callback callback = [this](pattern::Matcher& m) {
+        auto op = m.get_match_root();
+        if (transformation_callback(op)) {
+            return false;
+        }
+
+        return transform(*context, m);
+    };
+
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        std::make_shared<pattern::op::Or>(OutputVector{fq, fq_with_operation}),
+        "MoveFakeQuantize");
+    this->register_matcher(m, callback);
+}
+
+bool MoveFakeQuantize::transform(TransformationContext& context, ngraph::pattern::Matcher& m) {
```

Comment:
if LPT `ConcatTransformation` doesn't handle this `Concat`, then `MoveFakeQuantizeTransformation` should not handle too. You need to make existing `Concat::canBeTransformed` static (for example: `Concat::canBeTransformedStatic`) and reuse it in the `MoveFakeQuantizeTransformation`. Don't forget negative tests cases in functional & plugin tests, please.